src/domain/schemas/document-schemas.js

```diff
@@ -83,6 +85,7 @@ export const documentDBSchema = joi.object({
   updatedOn: joi.date().required(),
   updatedBy: idOrKeySchema.required(),
   title: joi.string().required(),
+  description: joi.string().allow('').max(1000).required(),
```

Comment:
What do you think about having this value in a const somewhere, using it here and also passing it to the translation?
We have other cases like this too (tag validation I think, probably password too), maybe we could change those as well in the future?

Reply:
True, good idea. I'll extract it.

Reply:
Extracted it (and a few others) into `validation-constants.js`

Reply:
Nice!